diffyscan/utils/constants.py

```diff
@@ -1,8 +1,52 @@
 import time
+import os
+import tempfile
 
 DIGEST_DIR = "digest"
 START_TIME = time.time()
 START_TIME_INT = int(START_TIME)
 DIFFS_DIR = f"{DIGEST_DIR}/{START_TIME_INT}/diffs"
 LOGS_PATH = f"{DIGEST_DIR}/{START_TIME_INT}/logs.txt"
 DEFAULT_CONFIG_PATH = "config.json"
+
+SOLC_DIR = os.path.join(tempfile.gettempdir(), "solc_builds")
+
+GITHUB_API_TOKEN = os.getenv("GITHUB_API_TOKEN", "")
+if not GITHUB_API_TOKEN:
+    raise ValueError("GITHUB_API_TOKEN variable is not set")
```

Comment:
Sorry, but it doesn't look like a compile-time constant
I would reconsider placing this env var somewhere else

Reply:
done